tools/dashboard-plugin-tests/cypress/integration/filterBar/filterBar.spec.ts

```diff
@@ -0,0 +1,21 @@
+// (C) 2019-2022 GoodData Corporation
+import { withTestConfig } from "../../tools/configure";
+import { PLUGIN_LATEST_CONFIG } from "../../support/constants";
+
+// customize.layout().customizeFluidLayout()
```

Comment:
Maybe forgetten code to be removed?

Reply:
Oh, I forgot to update the comment, it should point relevant tested API - updated